lib/init.ts

```diff
@@ -84,7 +84,7 @@ async function hasDirectory(dir) {
   }
 }
 
-function getDir(appName) {
-  const slug = slugify(appName, { lower: true });
+function getDir(ACCOUNT_ID) {
```

Comment:
if `ACCOUNT_ID` is a constant and we want to use that instead, that is fine. We should not have an argument to a function use this casing though.

Reply:
should either be changed to `accountId` or this function should take no parameters if the constant should always be used